provider/resources.go

```diff
@@ -633,12 +634,21 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 	// if we skipCredentialsValidation then we don't need to do anything in
 	// preConfigureCallback as this is an explicit operation
 	if skipCredentialsValidation {
+		log.Printf("[INFO] pulumi-aws: skip credentials validation")
 		return nil
 	}
 
 	var err error
 	credentialsValidationOnce.Do(func() {
+		log.Printf("[INFO] pulumi-aws: starting to validate credentials. " +
```

Comment:
@VenelinMartinov adding these logs around credentials validation; log module should add timestamps I think.